p2p/test/webrtc/webrtc_test.go

```diff
@@ -0,0 +1,49 @@
+package webrtc_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	libp2pwebrtc "github.com/libp2p/go-libp2p/p2p/transport/webrtc"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWebRTCStream(t *testing.T) {
```

Comment:
@marten-seemann added your file here, which runs without errors. I suppose you were testing more, as this is pretty basic? Anything particular you want to test more here (high level wise)?

As this does run out of the box.

Reply:
It doesn't on my machine. I assume 99ba632 is how you fixed this?